services/src/main/java/org/fao/geonet/api/userfeedback/UserFeedbackAPI.java

```diff
@@ -420,10 +463,24 @@ public ResponseEntity newUserFeedback(
 
             final UserSession session = ApiUtils.getUserSession(httpSession);
 
+            Locale locale = languageUtils.parseAcceptLanguage(request.getLocales());
+            ResourceBundle messages = ResourceBundle.getBundle("org.fao.geonet.api.Messages", locale);
+
             Log.debug("org.fao.geonet.api.userfeedback.UserFeedback", "newUserFeedback");
 
             final IUserFeedbackService userFeedbackService = getUserFeedbackService();
 
+            boolean recaptchaEnabled = settingManager.getValueAsBool(Settings.SYSTEM_USERSELFREGISTRATION_RECAPTCHA_ENABLE);
```

Comment:
With the recaptcha the check is done even if the option is not activated. I got this error on javascript console:
`Error: No reCAPTCHA clients exist. Nr@https://www.gstatic.com/recaptcha/api2/v1524685466525/recaptcha__en.js:406:411 Pr@https://www.gstatic.com/recaptcha/api2/v1524685466525/recaptcha__en.js:405:62 reload@http://localhost:8080/geonetwork/static/lib3d.js:153714:1732 link/c.submitForm/<@http://localhost:8080/geonetwork/static/gn_search_default.js:999:390 m/h.success/<@http://localhost:8080/geonetwork/static/lib3d.js:641:436 e/<@http://localhost:8080/geonetwork/static/lib3d.js:675:69 $eval@http://localhost:8080/geonetwork/static/lib3d.js:689:269 $digest@http://localhost:8080/geonetwork/static/lib3d.js:686:361 $apply@http://localhost:8080/geonetwork/static/lib3d.js:690:33`

Reply:
Fixed.